app/src/main/res/layout/activity_photo.xml

```diff
@@ -0,0 +1,51 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+    android:orientation="vertical"
+    android:background="@color/white">
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:scaleType="fitStart"
```

Comment:
![photo_view_width](https://cloud.githubusercontent.com/assets/6936968/26754516/fe3a6564-4899-11e7-8a7c-ce4930072884.png)

Is `scaleType` causing this?